src/components/Blasts/BlastVisual.js

```diff
@@ -204,13 +204,19 @@ const styles = theme => ({
   }
 });
 
+function CurrentDate() {
+  const date = new Date();
+  const today = date.toLocaleTimeString();
+  return today;
+}
```

Comment:
I would move this function inside the `BlastVisual` component or make a utils file for it.